### PR TITLE
documenting remember me

### PR DESCRIPTION
**Pull request by nguyenk**

### CODE REVIEW

File: doc/integrate_remember_me.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Intergrate the *UserService.RememberMe* feature
+------------------------------------
+
+Branch 2.1+ of the [UserService](http://mouf-php.com/packages/mouf/security.userservice/README.md) provide a cookie based remember me feature. If you want to plug it to your SimpleLoginController.
+
+This isn't a deep integration. You juste have to edit your `SimpleLoginView` instance, and set the `$addRememberMeCheckbox` property to 'true', and optionnaly provide a `$rememberMeLabel` :
```
Comment (moufmouf) [inline doc/integrate_remember_me.md]: "juste" => "just"
"optionnaly" => "optionally"

File: doc/integrate_remember_me.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+![remember me in SimpleLoginView](images/rememberMe.png)
+
+Note : Keep in ming that you have to configure the "Rememer Me" feature in the userService : [UserService - Activate the **Remember Me** feature (through cookie)](http://mouf-php.com/packages/mouf/security.userservice/doc/remember_me.md)
```
Comment (moufmouf) [inline doc/integrate_remember_me.md]: "ming" => "mind"

File: doc/integrate_remember_me.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Intergrate the *UserService.RememberMe* feature
+------------------------------------
+
+Branch 2.1+ of the [UserService](http://mouf-php.com/packages/mouf/security.userservice/README.md) provide a cookie based remember me feature. If you want to plug it to your SimpleLoginController.
```
Comment (moufmouf) [inline doc/integrate_remember_me.md]: > If you want to plug it to your SimpleLoginController.

Il doit manquer un bout de phrase?

**Comment by nguyenk** [comment]
Done :)

**Pull request merged**